templates/index.html

```diff
@@ -628,7 +619,7 @@ <h2 class="u-sv3">Run Android in the cloud with Anbox cloud</h2>
       <h2>Tell us what you have in mind</h2>
       <p>Anbox Cloud is a platform which offers new ways to deliver mobile apps. You can test it out in the cloud in a few clicks!</p>
       <p>If you'd like to chat or learn more about how we can help accelerate your company, please reach out to us here!</p>
-      <a class="p-button--positive" href="/contact-us">Talk to an expert</a>
+      <a class="p-button js-invoke-modal" href="/contact-us">Talk to an expert</a>
```

Comment:
Here as well, I'd keep this one positive to have a very visible call to action.

Reply:
We cannot have more than one positive button per page.